pkg/action/install.go

```diff
@@ -776,6 +777,16 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 		name = chartURL
 	}
 
+	if c.ReadOnlyFS {
+		if settings.Debug {
+			log.Output(2, fmt.Sprintf("[debug] Downloading %s to inMemoryCache\n", name))
```

Comment:
Could we refactor and use https://github.com/helm/helm/blob/7f3339cb4eec9800cb09a46512050fb509480f7e/cmd/helm/helm.go#L55  instead of duplicating? or add the same function in `action` package?

Reply:
This is used only in one place so defining a function would increase the symbol footprint unnecessarilty I feel.